tools/orca/README.md

```diff
@@ -0,0 +1,58 @@
+# Orca - A Kanidm Load Testing Tool
+
+Make a profile
+
+```
+orca setup-wizard --idm-admin-password ... \
+  --admin-password ... \
+  --control-uri 'https://localhost:8443' \
+  --profile ./profile.json
+```
+
+Test the connection
+
+```
+orca conntest --profile ./profile.json
+```
+
+Generate a State File
+
+```
+orca generate --profile ./profile.json --state ./state.json
```

Comment:
state does use json, maybe keep it consistent ?

Reply:
profile should be human editable and readable, state shouldn't.